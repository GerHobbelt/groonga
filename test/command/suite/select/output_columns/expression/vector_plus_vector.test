table_create Entries TABLE_NO_KEY
column_create Entries tags1 COLUMN_VECTOR ShortText
column_create Entries tags2 COLUMN_VECTOR ShortText

load --table Entries
[
{"tags1": ["A", "B"], "tags2": ["C", "D"]}
]

select Entries \
  --output_columns '_id, tags1 + tags2' \
 --command_version 3
