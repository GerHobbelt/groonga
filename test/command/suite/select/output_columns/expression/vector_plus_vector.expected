table_create Entries TABLE_NO_KEY
[[0,0.0,0.0],true]
column_create Entries tags1 COLUMN_VECTOR ShortText
[[0,0.0,0.0],true]
column_create Entries tags2 COLUMN_VECTOR ShortText
[[0,0.0,0.0],true]
load --table Entries
[
{"tags1": ["A", "B"], "tags2": ["C", "D"]}
]
[[0,0.0,0.0],1]
select Entries   --output_columns '_id, tags1 + tags2'  --command_version 3
{
  "header": {
    "return_code": 0,
    "start_time": 0.0,
    "elapsed_time": 0.0
  },
  "body": {
    "n_hits": 1,
    "columns": [
      {
        "name": "_id",
        "type": "UInt32"
      },
      {
        "name": "(tags1 + tags2)",
        "type": "ShortText"
      }
    ],
    "records": [
      [
        1,
        [
          "A",
          "B",
          "C",
          "D"
        ]
      ]
    ]
  }
}
